sootup.core/src/main/java/sootup/core/typehierarchy/MethodDispatchResolver.java

```diff
@@ -208,18 +212,16 @@ public static Optional<MethodSignature> resolveConcreteDispatch(
           // A not implemented method of an abstract class results into an abstract method
           return Optional.empty();
         }
-        // found method is abstract and the startclass is not abstract
+        // found method is abstract and the startClass is not abstract
         throw new ResolveException(
             "Could not find concrete method for " + m + " because the method is abstract");
```

Comment:
I think we should log the current ResolveException here and return Optional.empty() as well

Reply:
I think so, since the method is already returning a Optional.

Reply:
or better change it to IllegalStateException as we should already have found an implemented method for it (as the startClass is not abstract or it is abstract and its superclass has a method that is abstract as well.. so an exception is correct.

Reply:
but when the startclass is already abstract we should return an Optional.empty() the same way as if it would *not* implement an abstract method from its superclass

Reply:
I think it's better keep it `ResolveException` to stay the same with other exceptions in the file.